--- a/ngx_rtmp.h
+++ b/ngx_rtmp.h
@@ -19,7 +19,7 @@


 #if (NGX_WIN32)
-typedef __int8              int8_t;
+// typedef __int8              int8_t;
 typedef unsigned __int8     uint8_t;
 #endif

--- a/ngx_rtmp_core_module.c
+++ b/ngx_rtmp_core_module.c
@@ -608,12 +608,12 @@ ngx_rtmp_core_listen(ngx_conf_t *cf, ngx_command_t *cmd, void *conf)
 
         if (ngx_strncmp(value[i].data, "ipv6only=o", 10) == 0) {
 #if (NGX_HAVE_INET6 && defined IPV6_V6ONLY)
-            struct sockaddr  *sa;
+            struct sockaddr  *soca;
             u_char            buf[NGX_SOCKADDR_STRLEN];
 
-            sa = (struct sockaddr *) ls->sockaddr;
+            soca = (struct sockaddr *) ls->sockaddr;
 
-            if (sa->sa_family == AF_INET6) {
+            if (soca->sa_family == AF_INET6) {
 
                 if (ngx_strcmp(&value[i].data[10], "n") == 0) {
                     ls->ipv6only = 1;
@@ -631,7 +631,7 @@ ngx_rtmp_core_listen(ngx_conf_t *cf, ngx_command_t *cmd, void *conf)
                 ls->bind = 1;
 
             } else {
-                len = ngx_sock_ntop(sa,
+                len = ngx_sock_ntop(soca,
 #if (nginx_version >= 1005003)
                                     ls->socklen,
 #endif
